x-pack/filebeat/tests/system/test_http_endpoint.py

```diff
@@ -168,7 +214,6 @@ def test_http_endpoint_malformed_json(self):
 
         filebeat.check_kill_and_wait()
 
-        print("response:", r.status_code, r.text)
```

Comment:
please keep the print statements here. It is the only way to debug a failing test. An assert in a test run test not print why it failed, only that it did fail. Nosetest (which is used to run the tests) captures the output and presents it in the log output only if the test has failed.